validator/src/main.rs

```diff
@@ -322,6 +335,15 @@ pub fn main() {
                 .value_name("UNIX DOMAIN SOCKET")
                 .help("Stream entries to this unix domain socket path")
         )
+        .arg(
+            Arg::with_name("mnemonic_stdin")
+                .long("mnemonic-stdin")
+                .value_name("KEYPAIR NAME")
+                .multiple(true)
+                .takes_value(true)
+                .possible_values(&[IDENTITY_KEYPAIR_NAME, STORAGE_KEYPAIR_NAME, VOTING_KEYPAIR_NAME])
+                .help("Securely input a mnemonic code and optional passphrase for a keypair"),
```

Comment:
OT:  I'm not sure the path the entropy, seed and private key take are page pinned, so could potentially be sniffed off swap

Reply:
something for the docs?  "ensure your machine has no swap"

Reply:
docs sounds good to me too. Do we have those in-progress somewhere? I can add a note to them, unless @t-nelson you want to take care of it

Reply:
I've got ya! https://github.com/solana-labs/solana/issues/7084#issuecomment-557373311 😉